src/lib.rs

```diff
@@ -328,180 +310,69 @@ pub enum SpanKind {
     CONSUMER,
 }
 
-#[derive(Clone, Debug, PartialEq)]
-pub struct AxiomEvent {
-    // see https://axiom.co/docs/query-data/traces#browse-traces-with-the-opentelemetry-app for required field names
-    // first level fields
+fn serialize_time<S>(time: &OffsetDateTime, serializer: S) -> Result<S::Ok, S::Error>
+where
+    S: Serializer,
+{
+    let s = time
+        .format(&time::format_description::well_known::Rfc3339)
+        .map_err(serde::ser::Error::custom)?;
+    serializer.serialize_str(&s)
+}
+
+#[derive(Clone, Debug, Serialize, PartialEq)]
+pub struct OtelField {
+    #[serde(skip_serializing_if = "Option::is_none")]
     pub span_id: Option<SpanId>,
+    #[serde(skip_serializing_if = "Option::is_none")]
     pub trace_id: Option<TraceId>,
+    #[serde(skip_serializing_if = "Option::is_none")]
     pub parent_span_id: Option<SpanId>,
     pub kind: &'static str,
     pub name: Cow<'static, str>,
+    #[serde(skip_serializing_if = "Option::is_none")]
     pub duration_ms: Option<u64>,
+    #[serde(rename = "_time", serialize_with = "serialize_time")]
```

Comment:
`time` comes with this
```suggestion
    #[serde(rename = "_time", with = "time::serde::rfc3339")]
```